### PR TITLE
plugins/netconf/ce.py: add a dispatch rpc function to send and receive original rpc-reply.

### PR DESCRIPTION
**Pull request by yuandongx**
##### SUMMARY

It is so important to send and receive some original RPC xml 
that we get some extra information.
So add a function 'dispatch_rpc' to do that.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

lib/ansible/plugins/netconf/ce.py
##### ADDITIONAL INFORMATION

lib/ansible/plugins/netconf/ce.py

```paste below
exmaple

<rpc message-id="103" xmlns="urn:ietf:params:xml:ns:netconf:base:1.0">
  <get-next xmlns="http://www.huawei.com/netconf/capability/base/1.0" set-id="1"> 
  </get-next>
</rpc>

<rpc message-id="101" xmlns="urn:ietf:params:xml:ns:netconf:base:1.0">
  <update xmlns="http://www.huawei.com/netconf/capability/base/1.0"/>
</rpc>
```
### And also the pr has validated many times on real devices befor it was ready for review.

### CODE REVIEW

**Comment by hwDCN** [comment]
shipit

**Comment by xuxiaowei0512** [comment]
bot_status

File: changelogs/fragments/62588-plugins-netconf-ce.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "plugins/netconf/ce.py: add a dispatch rpc function to send and receive original rpc-reply"
```
Comment (felixfontein) [inline changelogs/fragments/62588-plugins-netconf-ce.yml]: I think the following is easier to understand:
```suggestion
  - "Cloudengine netconf plugin - add a dispatch RPC function to send and receive original RPC reply"
```

**Review by felixfontein — CHANGES_REQUESTED** [review]
I don't understand how this is a bugfix. It adds a new method, which is apparently not used by any code. You should really explain in the PR why this is a bugfix and what bug this fixes.

File: changelogs/fragments/62588-plugins-netconf-ce.yml (hunk 1 above)
Comment (yuandongx) [reply changelogs/fragments/62588-plugins-netconf-ce.yml]: integrated this PR into ansible/ansible#62587

**Pull request closed without merging**